blocks/http/test/qa_HttpBlock.cpp

```diff
@@ -0,0 +1,215 @@
+#include <future>
+
+#include <boost/ut.hpp>
+
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+
+#include <gnuradio-4.0/basic/common_blocks.hpp>
+#include <gnuradio-4.0/Block.hpp>
+#include <gnuradio-4.0/Buffer.hpp>
+#include <gnuradio-4.0/Graph.hpp>
+#include <gnuradio-4.0/reflection.hpp>
+#include <gnuradio-4.0/Scheduler.hpp>
+
+#include <gnuradio-4.0/http/HttpBlock.hpp>
+
+template<typename T>
+struct fixed_source : public gr::Block<fixed_source<T>, gr::PortOutNamed<T, "out">> {
+    using super_t = gr::Block<fixed_source<T>, gr::PortOutNamed<T, "out">>;
+    T value = 1;
+
+    gr::work::Result
+    work(std::size_t requested_work) {
```

Comment:
Can it be implemented using `processOne` or `processBulk` function?

Reply:
Done.